environments/staging/postgresql.yml

```diff
@@ -1,5 +1,7 @@
 override:
   pgbouncer_pool_mode: transaction
+  pgbouncer_max_connections: 400
```

Comment:
with 300 pg connections this could be much higher - like 1000 - though I doubt that's necessary for staging